src/jio-navbar.css

```diff
@@ -191,7 +191,7 @@ button.nav-link {
   }
 }
 
-@media (max-width: 991px) {
+@media (max-width: 1309px) {
```

Comment:
Is there a reason for the difference of one pixel between these two values?

Reply:
@Vandit1604 Do you know why?

Reply:
Oops! Sorry interchanged the values of max and min

Reply:
`@media (min-width: 1310px) {...}`

`@media (max-width: 1309px) {...}`

think need to change this? What you think @lemeurherve 

Also, there is a pixel diff in the initial one. So, I thought that it would have a specific reason and maintained the same pixel diff in this

Reply:
If having the same value doesn't invalid the fix, I would set the same value on both.

Reply:
Nope. the same value doesn't invalid the fix